Tests/SwerverTests/ActionSpec/NotAllowedActionSpec.swift

```diff
@@ -0,0 +1,23 @@
+import Quick
+import Nimble
+import Swerver
+import Foundation
+
+class NotAllowedActionSpec: QuickSpec {
+    
+    override func spec() {
+        describe("A Not Allowed action") {
+            let listOfAllowedMethods = ["GET", "POST"]
```

Comment:
I would avoid creating objects outside the scope of a test. I believe I made this comment in a different PR, but it was probably at the same time as you were already working on this new code.

Reply:
fixed